### DISCUSSION TITLE
Elapsed Time Format

### DISCUSSION THREAD

**Discussion by its-rakib** [The-OpenROAD-Project/OpenROAD-flow-scripts#864] [General] [answered]
![image](https://user-images.githubusercontent.com/127094304/224974088-af1d55e4-efb8-4fc0-a253-4ddb9373c34b.png)

I can't figure out the timing format here.
Does it say 27 min 43 second?  Can anyone elaborate further?

**Comment by vijayank88** [comment, marked as answer]
yes it is 27Mins:43Sec

**Reply by vijayank88** [reply]
Another example:
```
[INFO DRT-0180] Post processing.
Elapsed time: 1:56:16[h:]min:sec. CPU time: user 131164.46 sys 2621.18 (1917%). Peak memory: 52583904KB.
```
`1Hr:56Min:15sec`
